plugins_src/primitives/wpc_thread_en.lang

```diff
@@ -0,0 +1,31 @@
+%% -*- mode:erlang; erlang-indent-level: 2 -*-
```

Comment:
Remove this file,  English files should not be checked in.

Reply:
Done!